lib/footloose.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
These files are not meant to be executable.
It's a matter of taste whether you add a `#!` or just do this:
```suggestion
# shellcheck shell=bash
```

Reply:
applied